tests/integration_test/src/nvf_test_driver.py

```diff
@@ -317,6 +318,10 @@ def run_event_sequence(self, event_sequence):
                 stats = self._get_stats(target=TargetType.SERVER, job_id=self.job_id)
                 # update run_state
                 changed, run_state = _update_run_state(stats=stats, run_state=run_state, job_run_status=job_run_status)
+                if changed:
+                    print("****************************************************************")
+                    print("********************Updated RUN STATE***************************")
+                    print(run_state)
```

Comment:
Should use the assert to check and verify the "run_state" data. Should avoid using "print".